bundles/org.aposin.gem.core/src/org/aposin/gem/core/utils/ExecUtils.java

```diff
@@ -53,11 +53,13 @@ private static boolean isWindows() {
      * Similar to {@link Runtime#exec(String[])}, but discarding stdout/stderr.
      * 
      * @param cmdArgs command and arguments tokenized and quoted if required.
+     * @param directory the process working directory.
      * @return running process.
      * @throws IOException if the command fails to launch
      */
-    public static Process exec(final List<String> cmdArgs) throws IOException {
+    public static Process exec(final List<String> cmdArgs, Path directory) throws IOException {

```

Comment:
Can you please keep a helper method passing `null` to this one, to avoid API changes? This might be used by other plug-ins and this will break them.